Monika After Story/game/script-apologies.rpy

```diff
@@ -0,0 +1,250 @@
+#Create an apology db for storing our times
+default persistent._mas_apology_time_db = {}
+
+#Create a generic apology db. We'll want to know how many times the player has apologized for mas_apology_reason
+#Allows us the ability to apply diminishing returns on affection for repeated use of the same apology
+default persistent._mas_apology_reason_use_db = {}
+
+init -10 python in mas_apology:
+    apology_db = {}
+
+
+#going to need this post ev_handler init
+init 20 python:
```

Comment:
Code/functions/vars inside functions aren't checked for existence until the function is run, so it's okay to have the init level before they are defined.

Which init level are you trying to be after?

There is a chance that we mignt need to run this in an update script one day, so it should try to be availlable before init level 10.